SU-HSDO/suhumsci
# hotfix: remove hb-stretch-vertical-linked-cards additions from shs-4652

# READY FOR REVIEW

## Summary
- Removes additional styles from SHS-4652 that were affecting linked cards with descriptor text.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Build PR and go to https://hs-traditional.suhumsci.loc/collections-cards-uniform-and-stretch-1 
2. Edit the layout for this page and verify the following utility classes wrap the card collection region `hb-raised-cards hb-raised-cards--uniform-height hb-stretch-vertical-linked-cards`
3. Verify the page looks like: 
![Screenshot from 2023-03-31 15-40-34](https://user-images.githubusercontent.com/369018/229225487-9c149392-223b-4233-a7a0-f3e457f47b3a.png)


and NOT like this:
![Screenshot from 2023-03-31 15-32-31](https://user-images.githubusercontent.com/369018/229224052-c0ca5d6e-3c6d-4f93-998d-b7bf8c624d9d.png)


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
